route_following_plugin/launch/route_following_plugin_launch.py

```diff
@@ -0,0 +1,62 @@
+# Copyright (C) 2022 LEIDOS.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+from ament_index_python import get_package_share_directory
+from launch import LaunchDescription
+from launch_ros.actions import ComposableNodeContainer
+from launch_ros.descriptions import ComposableNode
+from launch.actions import DeclareLaunchArgument
+from launch.substitutions import LaunchConfiguration
+from carma_ros2_utils.launch.get_current_namespace import GetCurrentNamespace
+
+import os
+
+
+'''
+This file is can be used to launch the CARMA pose_to_tf.
+  Though in carma-platform it may be launched directly from the base launch file.
+'''
+
+def generate_launch_description():
+
+    # Declare the log_level launch argument
+    log_level = LaunchConfiguration('log_level')
+    declare_log_level_arg = DeclareLaunchArgument(
+        name ='log_level', default_value='WARN')
+        
+    # Launch node(s) in a carma container to allow logging to be configured
+    container = ComposableNodeContainer(
+        package='carma_ros2_utils',
+        name='pose_to_tf_container',
```

Comment:
This is not resolved, and since the place it has to be launched changed. First, please delete from https://github.com/usdot-fhwa-stol/carma-platform/blob/feature/route_following_plugin/carma/launch/guidance.launch#L135-L140

and add it in `carma-platform/carma/launch/plugins.launch.py` like how `inlanecruising_plugin` is launched. 
https://github.com/usdot-fhwa-stol/carma-platform/blob/feature/route_following_plugin/carma/launch/plugins.launch.py#L58-L78